src/main/java/com/google/sps/servlets/ReviewServlet.java

```diff
@@ -51,13 +52,25 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
Add comments on all function and class headers. 106 taught you better than this :(

Reply:
Meant to push the comments before the day ended... :(